Src/Model/Csv.php

```diff
@@ -1,38 +1,54 @@
 <?php
-
 namespace Aot\Model;
 
 
 class Csv
 {
     private $path;
 
-    public function __construct()
+    public function __construct(){
+        $this->path = ROOT . 'Source/testtakers.csv';
+    }
+
+    /**
+     * define the method
+     * @param $method
+     * @param null $id
+     * @return array
+     * @throws \Exception
+     */
+    public function csvMethod($method, $id = null)
     {
-        $this->path = ROOT . '/Source/testtakers.csv';
+        if ($method == 'show' && $id) {
+            return $this->getOne($id);
+        } elseif ($method == 'show' && $id == null) {
+            return $this->getAll();
+        }
     }
 
     /**
      * get all users
      * @return array
      */
-    public function getAll()
+    private function getAll()
     {
         return $this->contentExtract();
     }
 
     /**
-     * get on user
-     * @param  array $setting
-     * @return array $datas
+     * get one user with the id
+     * @param  array $id
+     * @return array $content
+     * @throws \Exception
      */
-    public function getOne($setting)
+    private function getOne($id)
```

Comment:
A method can be `public`, `protected` or `private`. If there is no purpose to protect it (as `private`) or no need to access it from outside, please change it as `protected`